steps/microsoft-teams-integration/1.2.1/step.yml

```diff
@@ -0,0 +1,145 @@
+title: Microsoft Teams Integration
+summary: Allows you to recieve build status updates on Microsoft Team using Incoming
+  Webhooks
+description: |
+  Uses Incoming Webhooks feature in Microsft Teams to send updates on your builds
+website: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+source_code_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+support_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration/issues
+published_at: 2021-02-06T21:58:08.132756+11:00
+source:
+  git: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration.git
+  commit: 9da9a2007cbaa7c79928ce52ba749248cb8c5bf3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: golang
+    bin_name: go
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: true
+inputs:
+- card_title: $BITRISE_APP_TITLE
+  opts:
+    description: |
+      Primary MessageCard title text.
+
+      If no value is given, the default value will be the `BITRISE_APP_TITLE`.
+    summary: The heading text at the top of the MessageCard
+    title: Card Title
+- enable_debug: false
```

Comment:
FYI, I suggest changing this to `"false"` to make sure it is parsed as a `string`, not a `bool`. Not a hard requirement now, but in the future we might require this.

Reply:
@zsolt-marta-bitrise As mentioned in the other change suggestion by Karol below, i'm going to close this and open a new PR with the Go dependency removed.

Can I leave the bool -> string change out for this patch update and include it in my next update (2.0.0)?

Reply:
Sure, that's totally fine ;)